ajax/libs/jquery.i18n/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "jquery.i18n",
+  "description": "jQuery.i18n is a jQuery based Javascript internationalization library. It helps you to internationalize your web applications easily.",
+  "filename": "jquery.i18n.js",
+  "keywords": [
+    "jquery",
+    "i18n",
+    "internationalization",
+	"l10n",
+	"localization"
```

Comment:
Let's use wrap the spaces here. Thanks.

Reply:
@extend1994 I do not see those spaces. Also fix tool does not change anything in the file
![image](https://user-images.githubusercontent.com/11975561/30266377-d57b3e68-96e7-11e7-93b0-e8cc7a65038f.png)

Reply:
@extend1994 I will try to commit again, but no manual changes were done by me, only fixtool was ran.